doc/releases/migration-guide-4.0.rst

```diff
@@ -288,5 +288,10 @@ Shell
 * ``kernel threads`` and ``kernel stacks`` shell command have been renamed to
   ``kernel thread list`` & ``kernel thread stacks``
 
+JWT (JSON Web Token)
+====================
+
+* Default signature library is now PSA Crypto APIs instead of Mbed TLS. (:github:`78243`)
```

Comment:
I think this could be a little bit more explanatory. The other option is not even necessarily Mbed TLS, it's either Mbed TLS or TinyCrypt depending on the signature algorithm. It could be nice also to mention the relevant Kconfig options that the user can enable/disable to have an influence on this.

Reply:
I kept it short (maybe too much) because I didn't want to duplicate the content of the release-notes. For a migration-guide I thought that simply listing the default behavior changes should be enough. After all the Konfigs to select ECDSA/RSA did not changed in this PR and RSA is still the default. What I added here is the possibility to choose between legacy or PSA APIs (but these new Kconfigs are already in the release-notes) and this is where the change happened.
So IMO a possible improvement of this comment is that PSA APIs are the default library that will be used for both RSA/ECDSA. New Kconfigs to switch back to legacy libraries are in the release-notes.
How does this sounds?

Reply:
RSA is still the default, but the previous (default) behavior of using Mbed TLS/TC is changed (to use PSA). I think it's good to mention enough information in the migration guide itself. You could simply add that to stick to the old (deprecated) behavior, explicitly enable `CONFIG_JWT_USE_LEGACY`.